src/submission/NavButton.jsx

```diff
@@ -34,7 +33,7 @@ const NavButton = ({ page, base, code, editsFetched, validationComplete, quality
       break
     case 'macro':
       suffix = 'submission'
-      if (preError || code === MACRO_EDITS) className = 'hidden'
+      if (preError || macroExists && !macroVerified) className = 'hidden'
```

Comment:
Clears ESLint mixed operators warning.

```suggestion
      if (preError || (macroExists && !macroVerified)) className = 'hidden'
```